src/components/DevconGrantsForm.js

```diff
@@ -427,7 +437,9 @@ const DevconGrantsForm = () => {
         />
         <ErrorDiv>
           {formState.eventDate.isTouched && !formState.eventDate.isValid && (
-            <RequiredError />
+            <div>
+              <ErrorMessage>Please provide a valid date in the future</ErrorMessage>
```

Comment:
I believe we'll want to configure this for translations (see other copy to see how this is done).